### PR TITLE
add gunicorn to application

### PR DESCRIPTION
**Pull request by ucfchandra**
Gunicorn added to application startup. Ignore odo files, they represent some attempts to get it working for local-dev.

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -24,12 +24,16 @@ RUN apt-get clean && \
...
 WORKDIR /app/
 
 RUN pip install -r requirements.txt
 RUN mv settings_local.templ.py settings_local.py
-RUN python manage.py makemigrations  
+RUN python3 manage.py makemigrations
+  
 
-CMD ["sh", "-c", "python3 manage.py migrate && python3 manage.py runserver 0.0.0.0:8000"]
+CMD ["sh", "-c", "python3 manage.py migrate && gunicorn -c config/gunicorn/dev.py"]
```
Comment (jmbarne3) [inline Containerfile:39]: The migrate command should be considered a prerequisite step that runs prior to the app running. As a matter of principle, the entry command should also represent the running of the app, and not any prerequisite steps in _building_ the app. I would recommend moving the `migrate` command up as a `RUN` directive, and leaving `gunicorn` as the only command on this line.

File: config/gunicorn/dev.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Restart workers when code changes (development only!)
+reload = False
+# Write access and error info to /var/log
+# accesslog = errorlog = "/var/log/gunicorn/dev.log"
+# Redirect stdout/stderr to log file
+capture_output = True
+# PID file so you can easily fetch process ID
+pidfile = "/var/run/gunicorn/dev.pid"
+# Daemonize the Gunicorn process (detach & enter background)
+daemon = False
```
Comment (jmbarne3) [inline config/gunicorn/dev.py:20]: Do we foresee having a separate wsgi file for each environment? How are we going to switch between files in the gunicorn command? Should we roadmap out a task to parameterize this file and pull the values from an `.env` file?

**Review by jmbarne3 — CHANGES_REQUESTED** [review]
Let's reorganize the container file per the comment, and maybe add the .odo directory to the `.gitignore` until we're really ready to integrate that tool fully into the project.

File: config/gunicorn/dev.py (hunk 2 above)
Comment (ucfchandra) [reply config/gunicorn/dev.py:20]: We could just change the CMD for each container in ECS to include the relevant parameters in the command. Most of these are probably not necessary. Can't see any reason why there would be differences between environments, but then again I don't have too much gunicorn experience.

**Review by jmbarne3 — APPROVED** [review]

**Pull request merged**